apps/docs/src/guide/getting-started/snippets/connecting-to-the-network.ts

```diff
@@ -1,12 +1,14 @@
 // #region main
 import { Provider, Wallet } from 'fuels';
 
-import { LOCAL_NETWORK_URL } from '../../../env';
+const NETWORK_URL = 'https://mainnet.fuel.network/v1/graphql';
+// const NETWORK_URL = 'https://testnet.fuel.network/v1/graphql';
+// const NETWORK_URL = 'https://localhost:<port>/v1/graphql';
```

Comment:
IMO the commented out lines looked a bit messy on the FE
```suggestion
const NETWORK_URL = 'https://mainnet.fuel.network/v1/graphql';
const TESTNET_NETWORK_URL = 'https://testnet.fuel.network/v1/graphql';
const LOCAL_NETWORK_URL = 'https://localhost:<port>/v1/graphql';
```

Reply:
The idea is that one could comment/uncomment the lines and re-run.